src/transactions/saga.ts

```diff
@@ -157,8 +167,7 @@ export function* sendAndMonitorTransaction<T>(
     return { receipt: txReceipt }
   } catch (error) {
     Logger.error(TAG + '@sendAndMonitorTransaction', `Error sending tx ${context.id}`, error)
-    yield* put(removeStandbyTransaction(context.id))
-    yield* put(transactionFailed(context.id))
+    yield* put(removeStandbyTransactionWithoutHash(context.id))
```

Comment:
hmm i think we need to distinguish between the 2 scenarios you mention - 
1. if the transaction has been mined and failed (i.e. the user has paid some gas fee) then we should show it on the tx feed. the intention is to explain to the user why their balance changed. This is less important on Celo since the gas fees are negligible, but is important for Ethereum.
2. if an error was thrown before the transaction made it to the blockchain, then we should remove the standby transaction since a user has not incurred any losses. in that case we should probably display an error banner or some kind of error, but we can add that later.

from that perspective, i don't think this is the correct change. it is indeed not a straightforward change to meet those 2 conditions so i'd recommend reverting this change and tackling it in a follow up PR?